docker/docker-compose.traefik-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+# docker-compose ingress overrides for traefik
+---
+version: "3.9"
+services:
+  web:
+    labels:
+      - "traefik.enable=true"
+      # Traefik will route requests with Host matching the SERVER_NAME environment variable (see .env)
+      - "traefik.http.routers.portal-${COMPOSE_PROJECT_NAME}.rule=Host(`${SERVER_NAME}`)"
+
+      - "traefik.http.routers.portal-${COMPOSE_PROJECT_NAME}.entrypoints=websecure"
+      - "traefik.http.routers.portal-${COMPOSE_PROJECT_NAME}.tls=true"
+      - "traefik.http.routers.portal-${COMPOSE_PROJECT_NAME}.tls.certresolver=letsencrypt"
+    networks:
+      - ingress
+      # TODO rename to internal
```

Comment:
why is this left as a TODO?  likely to break current deploys?

Reply:
> why is this left as a TODO? likely to break current deploys?

Yup, we'll need to bring the docker-compose project down before making this change (otherwise we'll have 2 running projects)

(By default, docker-compose uses `default` as the network name)

Reply:
Makes sense, sounds challenging to schedule w/ all of our deploys.

Reply:
The easiest way to roll that change out is to: take a backup, wipe the dc project, deploy the change, and restore the DB. 99% of that process is scripted, so restoring the psql dump would be the largest contributor to downtime (disk IO).

I regularly do this with eproms-stage, and it doesn't take more than a minute to load the prod psql dump